components/Breadcrumb/src/index.scss

```diff
@@ -30,10 +29,24 @@ ol.denhaag-breadcrumb__list {
   padding-inline-start: 0;
 }
 
-.denhaag-breadcrumb__item,
-.denhaag-breadcrumb__link {
+.denhaag-breadcrumb__item {
   align-items: center;
-  display: flex;
+  display: var(--denhaag-breadcrumb-item-display);
+}
+
+.denhaag-breadcrumb__item:nth-last-child(2) {
+  --denhaag-breadcrumb-item-display: flex;
+}
+
+.denhaag-breadcrumb__item:nth-last-child(2) > .denhaag-breadcrumb__link::before {
+  background: var(--denhaag-breadcrumb-link-color);
+  clip-path: path(
```

Comment:
I don't understand why we need to use `clip-path` is necessary for the breadcrumb design.
We're only using text and arrow svg's, no pictures.

Reply:
Well if you want to rewrite the whole breadcrumb component, be my guest. The reason why this is a `clip-path` is because on only the second-last the arrow-icon must be shown. Without creating a massive overhead in CSS and a bigger DOM, with the deadline in perspective.

Currently the breadcrumb-items are allowed in the following 2 posibilities:
1. SVG - (hidden) label - SVG;
2. label - SVG;

As for what you're requesting is to rewrite the whole breadcrumb styles and HTML to the following:
1. SVG - SVG - (hidden) label - SVG;
2. SVG - label - SVG;

Meanwhile a single clip-path can reduce a lot of nearly duplicate css.